test/test.js

```diff
@@ -145,13 +155,13 @@ describe('moo lexer', () => {
 
   test('accepts rules in an array', () => {
     const lexer = compile([
-      ['keyword', 'bob'],
+      ['keyword', 'Bob'],
```

Comment:
This is a fairly subtle change to a different test that tests something significant related to keywords. I would make it more explicit by putting it in its own test.